freqtrade/strategy/interface.py

```diff
@@ -429,13 +436,13 @@ def get_signal(self, pair: str, timeframe: str, dataframe: DataFrame) -> Tuple[b
         :param pair: pair in format ANT/BTC
         :param timeframe: timeframe to use
         :param dataframe: Analyzed dataframe to get signal from.
-        :return: (Buy, Sell) A bool-tuple indicating buy/sell signal
+        :return: (Buy, Sell, Date) A tuple indicating buy/sell signal for the latest date
```

Comment:
i assume there is an idea behind ... ?
but to be fair - the return value did not change (yet) ?

Reply:
i was adding that to pass the date (to get_stake_amount) also during live but then removed, because since get_signal is only called by the bot during live, it could already have the date as an argument, with a now() date and get_signal would then only locate the correct index (and maybe handle edge cases for duplicates)